src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -5116,5 +5116,69 @@ static void M()
         }
 
         #endregion
+
+
+        [Fact, WorkItem(4370, "https://github.com/dotnet/roslyn/issues/4370")]
+        public void HeadingHiddenSequencePointsPickUpDocumentFromVisibleSequencePoint()
+        {
+            var source = 
+@"#line 1 ""C:\Async.cs""
```

Comment:
Is the file path interpreted on this code path or just used literally in the PDB?  If it's interpreted do we need to limit this fix to windows only?